pkg/ccl/streamingccl/streamingest/stream_ingestion_job.go

```diff
@@ -248,30 +252,25 @@ func ingest(ctx context.Context, execCtx sql.JobExecContext, ingestionJob *jobs.
 	}
 	if reverted {
 		log.Infof(ctx, "job completed cutover on resume")
-		return nil
+		return completeIngestion(ctx, execCtx, ingestionJob)
 	}
 	if knobs := execCtx.ExecCfg().StreamingTestingKnobs; knobs != nil && knobs.BeforeIngestionStart != nil {
 		if err := knobs.BeforeIngestionStart(ctx); err != nil {
 			return err
 		}
 	}
-	client, err := connectToActiveClient(ctx, ingestionJob, execCtx.ExecCfg().InternalDB)
-	if err != nil {
-		return err
-	}
-	defer func() {
-		if err := client.Close(ctx); err != nil {
-			log.Warningf(ctx, "stream ingestion client did not shut down properly: %s", err.Error())
-		}
-	}()
-	if err = startDistIngestion(ctx, execCtx, ingestionJob, client); err != nil {
+
+	if err = startDistIngestion(ctx, execCtx, ingestionJob); err != nil {
 		return err
 	}
 	// A nil error is only possible if the job was signaled to cutover and the
 	// processors shut down gracefully, i.e stopped ingesting any additional
 	// events from the replication stream. At this point it is safe to revert to
 	// the cutoff time to leave the cluster in a consistent state.
-	return completeIngestion(ctx, execCtx, ingestionJob, client)
+	if err := revertToCutoverTimestamp(ctx, execCtx, ingestionJob); err != nil {
+		return err
+	}
+	return completeIngestion(ctx, execCtx, ingestionJob)
```

Comment:
should this comment stay with `completeIngestion`?

Reply:
done.

Reply:
I don't see a change here? I meant, should lines 270-272 move 5 lines up?